File: androidApp/src/main/java/com/medhelp/medhelp/data/model/settings/SimpleResponseBoolean.java
package com.medhelp.medhelp.data.model.settings;

import com.google.gson.annotations.SerializedName;

public class SimpleResponseBoolean {
    @SerializedName("imgError")
    private boolean error;

    @SerializedName("Message")
    private String message;

    @SerializedName("response")
    private Boolean response ;


    public boolean isError() {
        return error;
    }

    public void setError(boolean error) {
        this.error = error;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Boolean getResponse() {
        return response;
    }

    public void setResponse(Boolean response) {
        this.response = response;
    }
}

File: shared/src/commonMain/kotlin/com/medhelp/newmedhelp/model/SimpleResponseBoolean.kt
package com.medhelp.newmedhelp.model

import com.medhelp.shared.model.UserResponse
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class SimpleResponseBoolean {
    @SerialName("error")
    var error = false
    @SerialName("message")
    var message: String? = null
    @SerialName("response")
    var response: Boolean=false
}